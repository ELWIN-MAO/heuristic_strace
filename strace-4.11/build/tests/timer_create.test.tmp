
mym_waitpid=103037

mym_waitpid=103037

mym_waitpid=103037

syscall_enter_mym_scno=59  sys_name=execve

mym_waitpid=103037

mym_waitpid=103037

syscall_exit_mym_scno=59  sys_name=execve

mym_waitpid=103037

syscall_enter_mym_scno=12  sys_name=brk

mym_waitpid=103037

syscall_exit_mym_scno=12  sys_name=brk

mym_waitpid=103037

syscall_enter_mym_scno=21  sys_name=access

mym_waitpid=103037

syscall_exit_mym_scno=21  sys_name=access

mym_waitpid=103037

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=103037

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=103037

syscall_enter_mym_scno=21  sys_name=access

mym_waitpid=103037

syscall_exit_mym_scno=21  sys_name=access

mym_waitpid=103037

syscall_enter_mym_scno=2  sys_name=open

mym_waitpid=103037

syscall_exit_mym_scno=2  sys_name=open

mym_waitpid=103037

syscall_enter_mym_scno=5  sys_name=fstat

mym_waitpid=103037

syscall_exit_mym_scno=5  sys_name=fstat

mym_waitpid=103037

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=103037

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=103037

syscall_enter_mym_scno=3  sys_name=close

mym_waitpid=103037

syscall_exit_mym_scno=3  sys_name=close

mym_waitpid=103037

syscall_enter_mym_scno=21  sys_name=access

mym_waitpid=103037

syscall_exit_mym_scno=21  sys_name=access

mym_waitpid=103037

syscall_enter_mym_scno=2  sys_name=open

mym_waitpid=103037

syscall_exit_mym_scno=2  sys_name=open

mym_waitpid=103037

syscall_enter_mym_scno=0  sys_name=read

mym_waitpid=103037

syscall_exit_mym_scno=0  sys_name=read

mym_waitpid=103037

syscall_enter_mym_scno=5  sys_name=fstat

mym_waitpid=103037

syscall_exit_mym_scno=5  sys_name=fstat

mym_waitpid=103037

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=103037

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=103037

syscall_enter_mym_scno=10  sys_name=mprotect

mym_waitpid=103037

syscall_exit_mym_scno=10  sys_name=mprotect

mym_waitpid=103037

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=103037

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=103037

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=103037

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=103037

syscall_enter_mym_scno=3  sys_name=close

mym_waitpid=103037

syscall_exit_mym_scno=3  sys_name=close

mym_waitpid=103037

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=103037

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=103037

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=103037

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=103037

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=103037

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=103037

syscall_enter_mym_scno=158  sys_name=arch_prctl

mym_waitpid=103037

syscall_exit_mym_scno=158  sys_name=arch_prctl

mym_waitpid=103037

syscall_enter_mym_scno=10  sys_name=mprotect

mym_waitpid=103037

syscall_exit_mym_scno=10  sys_name=mprotect

mym_waitpid=103037

syscall_enter_mym_scno=10  sys_name=mprotect

mym_waitpid=103037

syscall_exit_mym_scno=10  sys_name=mprotect

mym_waitpid=103037

syscall_enter_mym_scno=10  sys_name=mprotect

mym_waitpid=103037

syscall_exit_mym_scno=10  sys_name=mprotect

mym_waitpid=103037

syscall_enter_mym_scno=11  sys_name=munmap

mym_waitpid=103037

syscall_exit_mym_scno=11  sys_name=munmap

mym_waitpid=103037

syscall_enter_mym_scno=222  sys_name=timer_create
timer_create(CLOCK_REALTIME, {sigev_value={int=-1159983635, ptr=0xdeadbeefbadc0ded}, sigev_signo=4207869677, sigev_notify=SIGEV_NONE}, 
mym_waitpid=103037

syscall_exit_mym_scno=222  sys_name=timer_create
[0]
mym_scall_rval=0
) = 0

mym_waitpid=103037

syscall_enter_mym_scno=5  sys_name=fstat

mym_waitpid=103037

syscall_exit_mym_scno=5  sys_name=fstat

mym_waitpid=103037

syscall_enter_mym_scno=12  sys_name=brk

mym_waitpid=103037

syscall_exit_mym_scno=12  sys_name=brk

mym_waitpid=103037

syscall_enter_mym_scno=12  sys_name=brk

mym_waitpid=103037

syscall_exit_mym_scno=12  sys_name=brk

mym_waitpid=103037

syscall_enter_mym_scno=222  sys_name=timer_create
timer_create(CLOCK_MONOTONIC, {sigev_value={int=-1159983635, ptr=0xdeadbeefbadc0ded}, sigev_signo=SIGALRM, sigev_notify=SIGEV_SIGNAL}, 
mym_waitpid=103037

syscall_exit_mym_scno=222  sys_name=timer_create
[1]
mym_scall_rval=0
) = 0

mym_waitpid=103037

syscall_enter_mym_scno=222  sys_name=timer_create
timer_create(CLOCK_REALTIME, {sigev_value={int=-1159983635, ptr=0xdeadbeefbadc0ded}, sigev_signo=SIGALRM, sigev_notify=SIGEV_THREAD, sigev_notify_function=0xdeadbeefbadc0ded, sigev_notify_attributes=0xcafef00dfacefeed}, 
mym_waitpid=103037

syscall_exit_mym_scno=222  sys_name=timer_create
[2]
mym_scall_rval=0
) = 0

mym_waitpid=103037

syscall_enter_mym_scno=39  sys_name=getpid

mym_waitpid=103037

syscall_exit_mym_scno=39  sys_name=getpid

mym_waitpid=103037

syscall_enter_mym_scno=222  sys_name=timer_create
timer_create(CLOCK_MONOTONIC, {sigev_value={int=-1159983635, ptr=0xdeadbeefbadc0ded}, sigev_signo=SIGALRM, sigev_notify=SIGEV_THREAD_ID, sigev_notify_thread_id=103037}, 
mym_waitpid=103037

syscall_exit_mym_scno=222  sys_name=timer_create
[3]
mym_scall_rval=0
) = 0

mym_waitpid=103037

syscall_enter_mym_scno=1  sys_name=write

mym_waitpid=103037

syscall_exit_mym_scno=1  sys_name=write

mym_waitpid=103037

syscall_enter_mym_scno=231  sys_name=exit_group

mym_waitpid=103037
+++ exited with 0 +++
