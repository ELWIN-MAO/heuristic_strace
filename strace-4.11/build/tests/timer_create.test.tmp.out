timer_create(CLOCK_REALTIME, {sigev_value={int=-1159983635, ptr=0xdeadbeefbadc0ded}, sigev_signo=4207869677, sigev_notify=SIGEV_NONE}, [0]) = 0
timer_create(CLOCK_MONOTONIC, {sigev_value={int=-1159983635, ptr=0xdeadbeefbadc0ded}, sigev_signo=SIGALRM, sigev_notify=SIGEV_SIGNAL}, [1]) = 0
timer_create(CLOCK_REALTIME, {sigev_value={int=-1159983635, ptr=0xdeadbeefbadc0ded}, sigev_signo=SIGALRM, sigev_notify=SIGEV_THREAD, sigev_notify_function=0xdeadbeefbadc0ded, sigev_notify_attributes=0xcafef00dfacefeed}, [2]) = 0
timer_create(CLOCK_MONOTONIC, {sigev_value={int=-1159983635, ptr=0xdeadbeefbadc0ded}, sigev_signo=SIGALRM, sigev_notify=SIGEV_THREAD_ID, sigev_notify_thread_id=103037}, [3]) = 0
+++ exited with 0 +++
lo IP Address 127.0.0.1
ens33 IP Address 192.168.159.137
IP Address 127.0.0.1
IP Address 192.168.159.137
